pkg/controller/gc.go

```diff
@@ -572,3 +576,34 @@ func (c *Controller) isOVNProvided(providerName string, pod *corev1.Pod) (bool,
 	}
 	return true, nil
 }
+
+func (c *Controller) getVmLsps() []string {
+	var vmLsps []string
+
+	if !c.config.EnableKeepVmIP {
+		return vmLsps
+	}
+
+	nss, err := c.namespacesLister.List(labels.Everything())
+	if err != nil {
+		klog.Errorf("failed to list namespaces, %v", err)
+		return vmLsps
+	}
+
+	for _, ns := range nss {
+		vms, err := c.config.KubevirtClient.VirtualMachine(ns.Name).List(&metav1.ListOptions{})
```

Comment:
Can we replace `ns.Name` with `""` to list resources in all namespaces? 
https://github.com/kubernetes/client-go/issues/159#issuecomment-288624475

Reply:
The apis from KubevirtClient can not  set namespace to "", the parameter is requested.
<img width="871" alt="image" src="https://user-images.githubusercontent.com/67130442/170404278-97591a67-b6cc-472d-b637-86a59dcb49d9.png">